openupgrade_scripts/scripts/mail/16.0.1.10/end-migration.py

```diff
@@ -0,0 +1,25 @@
+from openupgradelib import openupgrade
+
+
+def _mail_channel_privacy(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE mail_channel
+        SET is_private = true
+        WHERE public = 'private'
```

Comment:
@duong77476 như này mới đúng chứ
bổ sung thêm comment giải thích lý do tại sao lại như vậy
dẫn link sang cửa hàng bán module này nhé
do repo này đang public, đang có nhiều dev dùng repo này, biết đâu lại bán được được module
```suggestion
        WHERE public = 'private' AND channel_type = 'channel'
```

Reply:
vâng a